src/Application/SaveMappings/SaveMappingsPresenter.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseRDF\Application\SaveMappings;
+
+use ProfessionalWiki\WikibaseRDF\Application\MappingList;
+
+interface SaveMappingsPresenter {
+
+	public function presentSuccess(): void;
+
+	public function presentInvalidMappings( MappingList $mappings ): void;
+
+	public function presentSaveFailed(): void;
```

Comment:
This might be slightly ambiguous, but refers to the repository persistance failing versus validation failing earlier (presentInvalidMappings).